strategies/changed/src/main/java/org/arquillian/smart/testing/vcs/git/GitChangesDetector.java

```diff
@@ -57,6 +57,24 @@ private File findRepoRoot(File currentDir) {
         return extractEntries(diffs, this.repoRoot);
     }
 
+    public List<String> getLocalTests(Set<String> files) {
+        final List<String> localTests = files.stream()
+            .filter(this::matchPatterns)
+            .map(file -> {
+                try {
+                    final File sourceFile = new File(repoRoot, file);
+                    return new ClassNameExtractor().extractFullyQualifiedName(sourceFile);
```

Comment:
I know that in the original code this was like this, but shouldn't be do `extractFullyQualifiedName`  a static method (Since it requires no context) so we shouldn't have to instantiate every time?